.github/workflows/rust-test.yml

```diff
@@ -0,0 +1,23 @@
+name:                   Rust backend test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable
+jobs:
+  check:
+    name:               Check
+    runs-on:            ubuntu-latest
+    steps:
+      - name:           Checkout sources
+        uses:           actions/checkout@v2
+        with:
+          fetch-depth:  50
+      - name:           Checkout submodules
+        shell:          bash
+        run:            git submodule update --init --recursive
```

Comment:
I don't think this is required, is it?

Reply:
Not yet, but then I'll just forget to add it.